.github/workflows/e2e-aes.yml

```diff
@@ -0,0 +1,172 @@
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+        with:
+          lfs: "true"
+
+      # - name: Login to Docker Hub
+      #   uses: docker/login-action@v1
+      #   with:
+      #     username: ${{ secrets.DOCKER_HUB_USERNAME }}
+      #     password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Set up Python version
+        uses: actions/setup-python@v3
+        with:
+          python-version: "3.8"
+          
+      - name: Set up python dependencies
+        working-directory: benchmarks/aes/python
+        run: |
+          python3 -m pip install --upgrade pip
+          python3 -m pip install wheel
+          python3 -m pip install setuptools==49.6.0
+          python3 -m pip install ez_setup
+          GRPC_PYTHON_BUILD_SYSTEM_ZLIB=true
+          python3 -m pip install -r requirements.txt
+
+      - name: Install protoc
+        run: |
+          PROTOC_ZIP=protoc-3.14.0-linux-x86_64
+          curl -OL https://github.com/protocolbuffers/protobuf/releases/download/v3.14.0/$PROTOC_ZIP.zip
+          unzip -o $PROTOC_ZIP.zip -d $HOME/.local
+          chmod +x $HOME/.local/bin/protoc
+          rm -f $PROTOC_ZIP.zip
+      
+      - name: Build
+        working-directory: benchmarks/aes
+        env:
+          GOPRIVATE_KEY: ${{ secrets.EASE_LAB_ACCESS_KEY }}
+        run: |
+          go get -u google.golang.org/protobuf/cmd/protoc-gen-go@v1.26
+          go get -u google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.1
+          chmod +x $(go env GOPATH)/bin/*
+          echo $PATH
+          export PATH="$PATH:$(go env GOPATH)/bin"
+          echo $PATH
+          make all_image
+
+  test-compose:
+    name: Test Docker Compose
+    needs: build-and-push
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+#         file: [dc-aes-go.yaml,dc-aes-python.yaml,dc-aes-nodejs.yaml,dc-aes-go-tracing.yaml,dc-aes-python-tracing.yaml,dc-aes-nodejs-tracing.yaml]
+        runtime: [go, nodejs, python]
+        include:
+          - file: dc-aes-go.yaml
+            port: 50051
+          - file: dc-aes-go-tracing.yaml
+            port: 50051
+          - file: dc-aes-python.yaml
+            port: 50051
+          - file: dc-aes-python-tracing.yaml
+            port: 50051
+          - file: dc-aes-nodejs.yaml
+            port: 50051
+          - file: dc-aes-nodejs-tracing.yaml
+            port: 50051
+
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+        with:
+          lfs: "true"
+
+      - name: start docker-compose benchmark
+        working-directory: benchmarks/aes/yamls/docker-compose
+        run: |
+          docker-compose -f dc-aes-${{ matrix.runtime }}.yaml up &> log_file &
```

Comment:
Can you comment line 89. That we now iterate over the files instead of the runtimes that we can test also the files with tracing enabled. Then change the docker compose command to something like this:
docker-compose -f ${{ matrix.file }} up &> log_file &